src/components/CountrySelection.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+
+import Countries from '../countries';
+
+/**
+ * Renders a button to select a new country, and the currently
+ * selected country if one's been set in session storage.
+ *
+ * If the user has a country preference in session storage, we
+ * update the button text to "Change country".
+ */
+const CountrySelection = ({}) => {
+  // Get the country code. e.g., "BG" for "Bulgaria"
+  const [country] = useState(sessionStorage.getItem('country_pref'));
+
+  // Default to Great Britain
+  const countryName = Countries.fromAlpha2Code(country || 'GB').name;
+  const countryEmoji = Countries.fromAlpha2Code(country || 'GB').emoji;
```

Comment:
Always look into the session storage first. Defaults to 'GB' which I could see being controversial. @ reviewers, wdyt?

Reply:
Cn we make the default be no country? The idea of a filter is that it reduces the overall data to a subset of more relevant information, but that is an optional feature which is only used when folks need it.

Reply:
Sure thing! Will fix this 👍